dogs.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Dogs</title>
+  </head>
+
+  <body>
+    <header class="main-header">
+      <img class="adoptadog-logo" src="images/adoptadog-logo.svg" alt="Adoptadog logo">
+      <h2>Seattle's premiere dog adoption agency</h2>
+      <img class="cart-icon" src="images/cart-icon.svg" alt="Shopping cart icon">
+      <p>$1234.56</p>
+      <a href="checkout.html">Checkout</a>
+    </header>
+
+    <nav class="main-nav">
+      <a href="index.html" target="_blank">Home  /</a>
+      <a href="dogs.html" target="_blank">Dogs  /</a>
+      <a href="blog.html" target="_blank">Blog</a>
+    </nav>
+
+    <h2>Here are all of our Dogs!</h2>
+
+    <section>
+      <article class="dog-card">
+        <img src="images/murphy-card.jpg" alt="Picture of Murphy, reddish-brown fur with white on snout">
+        <h3>Murphy</h3>
+        <p><b>Cost to Adopt:</b> $123.45</p>
+        <p>Doggo ipsum pupperino extremely cuuuuuute heckin clouds big ol shoober smol</p>
+        <button type="button" name="adoptButton">Adopt</button>
+      </article>
+      <article class="dog-card">
+        <img src="images/poppy-card.jpg" alt="Picture of Poppy trotting down street, white and curly fur">
+        <h3>Poppy</h3>
+        <p><b>Cost to Adopt:</b> $123.45</p>
```

Comment:
Maybe consider using strong instead of the b tag, you can set the font weight in css